### ISSUE TITLE
"WiFi is disabled.." message while tethering

### ISSUE THREAD

**Issue by chr15m** [hypeapps/Endoscope#34]
If you tether your laptop to the Android device via WiFi hotspot or USB (which creates a network) the application will not let you stream.

**Comment by stfbee** [comment]
As far as i know, tethering in android doesn't allow clients to transfer data between itself and android device

**Comment by chr15m** [comment]
Yes it does. Your laptop gets an IP in the range 192.168.42.XXX and this is a normal network connection.
